client/build-tailwind.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Check for the existence of the Tailwind binary
+if [ ! -f ./node_modules/.bin/tailwindcss ]; then
+    echo "Tailwind CSS not found! Please run npm install inside the client directory."
+    exit 1
```

Comment:
Does this exit 1 make the build fail?

Reply:
```
std::process::ExitStatus
pub fn success(&self) -> bool
Was termination successful? Signal termination is not considered a success, and success is defined as a zero exit status
```

Reply:
So it would terminate the current process